pkg/pod/pod.go

```diff
@@ -202,6 +202,27 @@ func (builder *Builder) Delete() (*Builder, error) {
 	return builder, nil
 }
 
+// IsReady check if pod reached the Ready condition.
+func (builder *Builder) IsReady() bool {
```

Comment:
We have ready method WaitUntilReady in this package.

You can just pass timeout 1 second and it'll validate if pod is ready.